src/Button.js

```diff
@@ -7,6 +7,8 @@ class Button extends React.Component {
         accent: PropTypes.bool,
         className: PropTypes.string,
         colored: PropTypes.bool,
+        component: PropTypes.any,
```

Comment:
Nope, `PropTypes.element` would be already rendered element, but we should allow to pass a string (for html tags), class (for react components) and soon function (for react 0.14 feature to allow just pass function).

So we actually need to use `oneOfType` with `string` and `func`, but I think we should not restrict it. React 0.14 will allow to use pure function instead of a real class and there's so much issues with different components out there that did not assume that possibility and it's just annoying.

Anyway react will warn users if they'll try to pass something totally wrong.